vue/src/components/LayerSelection.vue

```diff
@@ -547,7 +547,7 @@ const addProposal = () => {
       :color="state.filters.addingSitePolygon ? 'primary' : ''"
       @click="addProposal()"
     >
-      <v-icon>mdi-plus</v-icon>Add Proposal
+      <v-icon>mdi-plus</v-icon>Add Proposal {{ scoringApp }}
```

Comment:
Debug leftover?
```suggestion
      <v-icon>mdi-plus</v-icon>Add Proposal
```

Reply:
Exactly, it was something from yesterday before the demo to remove the Add Proposal from the scoring view.  I've updated it.